src/apps/lwaftr/lwutil.lua

```diff
@@ -69,3 +69,8 @@ end
 function is_ipv4(pkt)
    return rd16(pkt.data + o_ethernet_ethertype) == n_ethertype_ipv4
 end
+
+-- This does not deal with vlans
```

Comment:
What would need to be handled for VLANs? VLAN is after the DST MAC address in the ethernet header so I don't see what would be different with VLANs.